pkg/debug/start_debug.go

```diff
@@ -41,11 +39,13 @@ func StartDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alte
 }
 
 func startDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alternateImageValue string) error {
-	deployment, err := verifyDeploymentExists(context, clusterNamespace, deploymentName)
+	OriginalDeployment, err := VerifyDeploymentExists(context, clusterNamespace, deploymentName)
```

Comment:
nit: local vars would be lower case
```suggestion
	originalDeployment, err := VerifyDeploymentExists(context, clusterNamespace, deploymentName)
```